api/src/interfaces/transaction.ts

```diff
@@ -1,5 +1,8 @@
 import { AddressOrPair, SignerOptions } from '@polkadot/api/types';
+import { Event } from '@polkadot/types/interfaces';
+import { ISubmittableResult } from '@polkadot/types/types';
+// export interface TransactionStatusCb {
+//   (event: Event): void | Promise<void>;
+// }
```

Comment:
Should it remain?

Reply:
Yeah, thank you. This file is unnecessary